foreman.te

```diff
@@ -350,6 +349,28 @@ optional_policy(`
     ')
 ')
 
+#######################################
+#
+# Remote Execution
+#
+
+# Allow Foreman to connect to Foreman Proxy on port 9090 (Katello)
```

Comment:
The placing here implies this is REX but it's core Foreman functionality.

Reply:
Good catch, I've moved this block accidentally actually.